src/index.html

```diff
@@ -1,17 +1,140 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <fieldset class="groups">
+      <legend>Personal information</legend>
+      <div class="form">
+        <label>
+          Surname:
+          <input type="text" name="Surname" required />
+        </label>
+      </div>
+      <div class="form">
+        <label>
+          Name:
+          <input type="text" pwname="Name" />
```

Comment:
write all names with first lowercase letter

```suggestion
          <input type="text" name="name" />
```